core/services/ardupilot_manager/rest_api.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python3
+from typing import Any, Dict, List
+
+import uvicorn
+from ArduPilotManager import ArduPilotManager
+from fastapi import FastAPI
+from mavlink_proxy.Endpoint import Endpoint, EndpointType
+
+app = FastAPI()
+autopilot = ArduPilotManager()
+
+
+@app.get("/available_endpoints", response_model=List[Dict[str, Any]])
+def get_available_endpoints() -> Any:
+    return [endpoint.__dict__ for endpoint in autopilot.get_endpoints()]
+
+
+@app.post("/endpoint", response_model=bool)
+def create_new_endpoint(connection_type: EndpointType, place: str, argument: str = "") -> Any:
+    if autopilot.add_endpoint(Endpoint({"connType": connection_type, "place": place, "argument": argument})):
+        return True
+    return False
+
+
+@app.delete("/endpoint", response_model=bool)
+def remove_endpoint(connection_type: EndpointType, place: str, argument: str = "") -> Any:
+    if autopilot.remove_endpoint(Endpoint({"connType": connection_type, "place": place, "argument": argument})):
+        return True
+    return False
+
+
+if __name__ == "__main__":
+    uvicorn.run("rest_api:app", host="0.0.0.0", port=8000)
```

Comment:
can we have a better name to identify the service ?

Reply:
Sure! Any ideia?

Reply:
ardupilot-manager ?

Reply:
Got the ideia.